src/models/curriculum.ts

```diff
@@ -1,5 +1,5 @@
 import { SnapshotIn, types } from "mobx-state-tree";
-import { AuthoredContentModel } from "./authored-content";
+import { DocumentContentModel } from "./document-content";
```

Comment:
Is the reasoning here that `Document` is more generic, since this could describe content loaded into a tab or a workspace? `Authored` is a little more descriptive to me.

Reply:
What came up in the discussion today is that the same content model should be used for the authored content and the user's content, i.e. it's a common document model.